eofconvention.fsx

```diff
@@ -0,0 +1,29 @@
+open System.IO
+
+let EOFIsEOL(path: FileInfo) = 
+    try
+        use sr = new StreamReader (path.FullName)
+        let filetext = sr.ReadToEnd()
+        (filetext |> Seq.last = '\n')
+        
+    with
+        | :? System.ArgumentException -> printfn "%A" path; true
+        
+
+let EOFIsNotEOL(path: FileInfo) = not (EOFIsEOL path)
```

Comment:
@realmarv no need to create this function, you can do this in the line that calls "filter" function (less code this way)

Reply:
Done